python/ray/data/context.py

```diff
@@ -98,9 +98,8 @@
 DEFAULT_AUTO_LOG_STATS = False
 
 # Whether to enable optimizer.
-DEFAULT_OPTIMIZER_ENABLED = bool(
-    int(os.environ.get("RAY_DATA_NEW_EXECUTION_OPTIMIZER", "0"))
-)
+DEFAULT_OPTIMIZER_ENABLED = True
+# DEFAULT_OPTIMIZER_ENABLED = False
```

Comment:
should we still keep the flag, but just set it to "1" by default?

Reply:
yeah, already did that.